opengrok-indexer/src/main/java/org/opengrok/indexer/util/Statistics.java

```diff
@@ -28,28 +28,47 @@
 import static org.opengrok.indexer.util.StringUtils.getReadableTime;
 
 public class Statistics {
-        
-  private final long startTime;  
 
-  public Statistics() {
+    private final long startTime;
```

Comment:
I still prefer using `Instant` and `Duration` classes for computing the duration as it is more high level. If you do not, then feel free to ignore.

Reply:
why not, changed. I like `Duration.between()`.